src/components/FocusTrap/FocusTrapForModal/FocusTrapForModalProps.ts

```diff
@@ -1,6 +1,9 @@
+import type {FocusTargetOrFalse} from 'focus-trap-react';
+
 type FocusTrapForModalProps = {
     children: React.ReactNode;
     active: boolean;
+    initialFocus?: FocusTargetOrFalse | undefined | (() => void);
```

Comment:
Can we avoid using `FocusTargetOrFalse` and use `HTMLElement | false` instead? 
i.e. something like this `initialFocus?: HTMLElement | false | undefined | (() => void);`